lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -117,6 +117,14 @@ sub ensure_size {
     return $size;
 }
 
+sub hidden {
+    # 1 if asset should not be linked from the web UI (as set by
+    # 'hide_asset_types' config setting), otherwise 0
+    my ($self) = @_;
+    my @types = split(/ /, $app->config->{global}->{hide_asset_types});
```

Comment:
What? Sorry, but I just have a really hard time following you. Do you mean `$OpenQA::Utils::app->config->{global}->{hide_asset_types}`?